packages/curriculum-helpers/lib/question-types.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  PRACTICE: 'practice',
+  REVISION: 'revision'
```

Comment:
So Quiz is also a type of question, I wonder if it should go here?

Reply:
hmm good question.

I needed these because I use the same compiler for revision and practice question and needed a way to differentiate the key at which they are added at in the final JSON

otherwise, I capture the quiz via section-names constant, and it has it's separate compiler because they are somehow different than the other questions through the additional `headline` field

Reply:
yeah but currently how quiz questions work in the EnkiMail, they won't have multiple gaps (actually no gaps at all)

think this is a fundamental separation between them